src/test/templates/sam/samTemplateGenerator.test.ts

```diff
@@ -36,12 +36,16 @@ describe('SamTemplateGenerator', () => {
         await del([tempFolder], { force: true })
     })
 
-    it('Produces a minimal template', async () => {
-        await new SamTemplateGenerator()
+    function makeMinimalTemplate(): SamTemplateGenerator {
+        return new SamTemplateGenerator()
             .withCodeUri(sampleCodeUriValue)
             .withFunctionHandler(sampleFunctionHandlerValue)
             .withRuntime(sampleRuntimeValue)
             .withResourceName(sampleResourceNameValue)
+    }
+
+    it('Produces a minimal template', async () => {
```

Comment:
`Produces a template with a MemorySize and Environment value`

Reply:
I had a series of PRs for these different fields (MemorySize and Environment). Now that both are committed, I'll expand this test out to verify each separately.